arrow/build.gradle.kts

```diff
@@ -36,6 +37,7 @@ dependencies {
 plugins.withType<JavaPlugin> {
     tasks {
         configureEach<Test> {
+            enabled = !OperatingSystem.current().isWindows
```

Comment:
It seems it is sufficient to exclude just the arrow tests, so I'd vote to go with this less drastic approach 👍  

Minor: perhaps we should add some comments, e.g.:
```
// Arrow test is disabled because it is not supported on Windows
// See https://arrow.apache.org/docs/java/install.html#system-compatibility
```

And since we are touching this file, maybe worth adding also some comments to the `jvmArgs` below:
```
// The following option `--add-opens=java.base/java.nio=ALL-UNNAMED` is required in jdk17+
// to avoid error. See https://arrow.apache.org/docs/java/install.html#java-compatibility
```

Reply:
Good idea. I added comments in https://github.com/apache/calcite/pull/3778/commits/c156046482bc929539fad70ff4a59dc64da219ed